src/funcTest/groovy/info/solidsoft/gradle/pitest/functional/AbstractPitestFunctionalSpec.groovy

```diff
@@ -1,9 +1,11 @@
 package info.solidsoft.gradle.pitest.functional
 
+import groovy.transform.CompileDynamic
 import nebula.test.IntegrationSpec
 import nebula.test.functional.ExecutionResult
 
-abstract class AbstractPitestFunctionalSpec extends IntegrationSpec {
+@CompileDynamic
+class AbstractPitestFunctionalSpec extends IntegrationSpec {
```

Comment:
That class shouldn't be treated as a separate test specification, even if there are - by mistake - any public methods. Coudn't it be stayed as `abstract`?